common/v2/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx

```diff
@@ -230,6 +235,9 @@ export default function WalletBreakdownView({
           ))}
         </BreakDownBalanceList>
         <BalanceTotalWrapper>
+          <ViewDetailsLink onClick={toggleShowChart}>
+            {translate('WALLET_BREAKDOWN_MORE')}
```

Comment:
View details link is already present but visible only when Other Tokens are visible. If you want to change the behavior please remove existing code (https://github.com/MyCryptoHQ/MyCrypto/blob/misc-design-fixes-2/common/v2/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx#L226) because right now this button could be rendered twice.

Screenshot:
![image](https://user-images.githubusercontent.com/12676806/64237270-4d91ec00-cefc-11e9-8b9b-0682dc82071d.png)